pytools/__init__.py

```diff
@@ -1597,6 +1597,32 @@ def github_markdown(self):
 
         return "\n".join(lines)
 
+    def csv(self, dialect='excel', **fmtparams):
```

Comment:
Maybe make `fmt_params` a separate argument? Otherwise we'll have a bear of a time extending this interface later.

Reply:
Fixed (if I understood your comment correctly).